src/using-workflow/configuring-an-application.md

```diff
@@ -80,12 +80,44 @@ appname to the old one:
 By default, Workflow only checks that your application containers start in
 their Pod. If you would like Kubernetes to respond to appliation health, you
 may add a health check by configuring URL, port, initial delay, and timeout
-values. The health checks are implemented as [Kubernetes container probes][kubernetes-probes].
+values.
+
+The health checks are implemented as [Kubernetes container probes][kubernetes-probes]. Currently only HTTP GET is supported.
+
+Available configuration options for health checks are the following, including defaults:
+
+**HEALTHCHECK_URL**
+
+Path in the application to use for health check, such as /health - This value needs to be set for any health check to happen. Needs to accept a HTTP GET request and return a HTTP status between 200-399.
```

Comment:
I changed it to `/healthz` but I would say tho that that spelling is a very k8s oriented, or perhaps it comes from other components it uses but never really seen it before touching k8s ;)